src/fides/api/models/detection_discovery.py

```diff
@@ -205,6 +206,9 @@ def derive_execution_trigger_dict(data: Dict[str, Any]) -> None:
         """
         execution_frequency = data.pop("execution_frequency", None)
         execution_start_date = data.pop("execution_start_date", None)
+        if execution_frequency == MonitorFrequency.NOT_SCHEDULED:
+            data["monitor_execution_trigger"] = None
+            return
```

Comment:
Maybe we can add a case to the parameters of the test `test_create_monitor_config_execution_trigger_logic` so that we cover this line , what do you think?

Reply:
actually I see you did this already, unsure why codecov's reporting that these lines aren't covered